test/integration/thermal_system.cc

```diff
@@ -76,20 +79,93 @@ TEST_F(ThermalTest, TemperatureComponent)
             // verify temperature data belongs to a visual
             EXPECT_NE(nullptr, _ecm.Component<components::Visual>(_id));
 
+            return true;
+          });
+
+      _ecm.Each<components::TemperatureRange, components::SourceFilePath,
+        components::Name>(
+          [&](const ignition::gazebo::Entity &_id,
+              const components::TemperatureRange *_tempRange,
+              const components::SourceFilePath *_heatSigURI,
+              const components::Name *_name) -> bool
+          {
+            // store temperature range data
+            entityTempRange[_name->Data()] = _tempRange->Data();
+
+            // store heat signature URI data
+            heatSignatures[_name->Data()] = _heatSigURI->Data();
+
+            // verify temperature range data belongs to a visual
+            EXPECT_NE(nullptr, _ecm.Component<components::Visual>(_id));
+
             return true;
           });
     });
   server.AddSystem(testSystem.systemPtr);
 
-  // verify nothing in map at beginning
+  // verify nothing in the maps at beginning
   EXPECT_TRUE(entityTemp.empty());
+  EXPECT_TRUE(entityTempRange.empty());
+  EXPECT_TRUE(heatSignatures.empty());
 
   // Run server
   server.Run(true, 1, false);
 
+  const std::string sphere_visual = "sphere_visual";
```

Comment:
consider using camelCase.

Reply:
resolved in https://github.com/ignitionrobotics/ign-gazebo/pull/498/commits/51c106da7e95445b64ac2ca071498773b6ea8e46